keyboards/kprepublic/bm65hsrgb_iso/keymaps/p4yne/keymap.c

```diff
@@ -0,0 +1,478 @@
+/* Copyright 2019 ashlar 2021 peepeetee, deadolus 2022 p4yne
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "keymap.h"
```

Comment:
I would recommend renaming the `keymap.h` file to prevent collisions with `quantum/keymap.h`.

Reply:
Most of the keymaps I looked at had the same pattern. keymap.c+keymap.h isn't c-style good practice to have those files share their name?

Reply:
@drashna Is the rename blocking the merge? Feedback is much appreciated. Thank you.

Reply:
The biggest issue is that `/quantum/keymap.h` exists. And that collision may cause serious issues.

Reply:
okay, thank you then I'll rename it this was not clear to me from your initial response.